chibuezeayogu/ecommerce-api
# App setup and configurations

**What does the PR do?**
- This PR generates and configures the application

**Description of Task to be completed?**
- Ensure application is setup and running.
- Ensure [Rubocop](https://github.com/rubocop-hq/rubocop-rails) is setup and configured properly
- Ensure [Rspec](https://github.com/rspec/rspec-rails) is setup and configured properly

**How should this be manually tested?**
- Clone the repository [Ecommerce-API](https://github.com/chibuezeayogu/Ecommerce-API.git) and
- Checkout to  this branch `chore-initial-setup`
- run `rails db:create` then `rails db:migrate` on the terminal and ensure it successfully executed
- Ensure `rubocop` and `rspec` commands runs successfully.
